tests/test_bs.py

```diff
@@ -263,6 +263,17 @@ def fin():
     return _setup_clients_in_namespace(testconfig['namespace'])
 
 
+@pytest.fixture(scope='module')
+def setup_network(request, init_session, setup_oracle, setup_poet,
+                  setup_bootstrap, setup_clients, add_curl, wait_genesis):
+
+    # This fixture deploy a complete Spacemesh network and returns only after genesis time is over
+    network_deployment = NetworkDeploymentInfo(dep_id=init_session,
```

Comment:
to add more nodes while the test is running we'll need the oracle and poet data aswell. maybe NetworkDeploymentInfo should include them.

Reply:
it does. see line number 267

Reply:
it does but you don't set them here.